data/src/main/java/com/example/data/backend/models/LoginErrorDto.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.data.backend.models
+
+import com.squareup.moshi.Json
+
+data class LoginErrorDto(
```

Comment:
Not Login error dto

its global errorResponse Type, we should have generic for this